model.js

```diff
@@ -5,4 +5,16 @@ exports.selectTopics = () => {
     return db.query("SELECT * FROM topics").then(({rows}) =>{
         return rows
     })
+}
+
+exports.selectArticleById = (id) =>{
+    return db.query(`SELECT * FROM articles WHERE article_id = ${id}`).then(({rows}) =>{
```

Comment:
Since you used the ${id} method here, the database is open to SQL injection. You will need to use the $1 syntax and an array to make sure this can't happen. You should refactor this.